src/query/procedure/callable_alias_mapper.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include "callable_alias_mapper.hpp"
+
+#include <algorithm>
+#include <array>
+#include <filesystem>
+#include <fstream>
+
+#include <spdlog/spdlog.h>
+#include <json/json.hpp>
+
+#include "utils/logging.hpp"
+
+namespace memgraph::query::procedure {
+
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
+CallableAliasMapper gCallableAliasMapper;
+
+void CallableAliasMapper::LoadMapping(const std::filesystem::path &path) {
+  using json = nlohmann::json;
+  if (!path.empty()) {
```

Comment:
```suggestion
  if (path.empty()) {
```

And add a unit test which covers the scenario of correctly and incorrectly loading the mapping